pkg/validate/container.go

```diff
@@ -143,6 +143,44 @@ var _ = framework.KubeDescribe("Container", func() {
 			verifyExecSyncContainOutput(rc, containerID, cmd)
 		})
 
+		It("runtime should support getting expected error when update resource of container with invalid memory limitation [Conformance]", func() {
+			By("create container")
+			containerID := framework.CreateDefaultContainer(rc, ic, podID, podConfig, "container-for-UpdateContainerResources-negative-test-")
+
+			By("start container")
+			startContainer(rc, containerID)
+
+			By("test UpdateContainerResources with invalid memory")
+			Resources := &runtimeapi.LinuxContainerResources{
+				MemoryLimitInBytes: 1024,
+			}
+			expectError := true
+			expectedErrStr := "Minimal memory should greater than 4M"
+			verifyUpdateContainerResources(rc, containerID, Resources, expectError, expectedErrStr)
+		})
+
+		It("runtime should support updating container resources [Conformance]", func() {
+			By("create container")
+			containerID := framework.CreateDefaultContainer(rc, ic, podID, podConfig, "container-for-UpdateContainerResources-test-")
+
+			By("start container")
+			startContainer(rc, containerID)
+
+			By("test UpdateContainerResources")
+			Resources := &runtimeapi.LinuxContainerResources{
+				CpuPeriod:          1000,
+				CpuQuota:           0,
+				CpuShares:          1024,
+				CpusetCpus:         "",
+				CpusetMems:         "",
+				MemoryLimitInBytes: 4194304,
+				OomScoreAdj:        0,
+			}
+			expectError := false
+			expectedErrStr := ""
+			verifyUpdateContainerResources(rc, containerID, Resources, expectError, expectedErrStr)
```

Comment:
We should check whether the value under `/sys/fs/cgroup` is meeting our expectation.

Reply:
sure, I have added cgroup related check in v2 patch.